js/ui/grid_core/ui.grid_core.focus.js

```diff
@@ -790,9 +790,12 @@ module.exports = {
                     return $(this.getRowElement(rowIndex >= 0 ? rowIndex : 0));
                 },
 
-                _scrollToElement: function(element) {
+                scrollElementTop: function($element) {
```

Comment:
Bad name. May be, scrollToRowElement?

Reply:
Have agreed rename to scrollToElementVertically